docs/03-tutorials/00-application-connectivity/ac-03-register-manage-services.md

```diff
@@ -2,275 +2,117 @@
 title: Register a service
 ---
 
-This guide shows you how to register a service of your external solution in Kyma.
+This guide shows you how to register a service of your external solution in Kyma. For this example, we use a Basic Authentication-secured API.   
 
-## Prerequisites
+>**NOTE:** Learn how to [register APIs secured with different security schemes or protected against cross-site request forgery (CSRF) attacks](ac-04-register-secured-api.md).
 
-- A valid certificate signed by the Kyma Certificate Authority
-- The [jq](https://stedolan.github.io/jq/download/) tool to prettify the JSON output
-- Your [Application name exported](ac-01-create-application.md#prerequisites) as an environment variable
-- Your [cluster domain, generated client certificate and key exported](ac-02-get-client-certificate.md#generate-a-csr-and-send-it-to-kyma) as environment variables
+## Prerequisites
 
-> **CAUTION:** On a local Kyma deployment, skip SSL certificate verification when making a `curl` call, by adding the `-k` flag to it. Alternatively, add the Kyma certificates to your local certificate storage on your machine using the `kyma import certs` command.
+Before you start, expose the following as environment variables:
+- Your [Application](./ac-01-create-application.md#prerequisites) name
+- Username and password to access the external system
+- Name of the Secret containing the service credentials
+- Name of your service
+- URL to your service
+- Unique ID identifying your service within the Application CR
+- Relative path in your service
+
+    ```bash
+    export APP_NAME=test-app
```

Comment:
```suggestion
    export APP_NAME={APPLICATION_NAME}
```

Reply:
Shouldn't we follow this pattern?

Reply:
No, 'cause we want it to be cop-paste ready. If someone wants to provide their own name, they can do so - the name is descriptive enough and line 30 tells the user to do so :)